applications/ShallowWaterApplication/custom_elements/conservative_element.cpp

```diff
@@ -145,11 +145,17 @@ void ConservativeElement<TNumNodes>::CalculateArtificialViscosity(
 {
     double jump = .0;
     array_1d<double,2> inner_grad_h = prod(rData.nodal_h + rData.nodal_z, rDN_DX);
-    for (auto& r_elem : this->GetValue(NEIGHBOUR_ELEMENTS)) {
```

Comment:
Please, wait a little before merging this branch, let me check if the behavior is unmodified. I will run some validation tests

Reply:
ping @miguelmaso

Reply:
sorry for the delay. I've just checked the results of the tests and it is OK